sdv/tabular/base.py

```diff
@@ -396,29 +396,28 @@ def _conditionally_sample_rows(self, dataframe, condition, transformed_condition
         if num_sampled_rows < num_rows:
             # Didn't get enough rows.
             if len(sampled_rows) == 0:
-                error = ('Unable to sample any rows for the given conditions: '
-                         f'`{transformed_condition}`. '
-                         f'Try increasing `max_tries` (currently: {max_tries}) or increasing '
-                         f'`batch_size_per_try` (currently: {batch_size_per_try}). Note that '
-                         'increasing these values will also increase the sampling time.')
-                raise ValueError(error)
-
-            elif not graceful_reject_sampling:
-                error = f'Could not get enough valid rows within {max_tries} trials.'
-                raise ValueError(error)
+                user_msg = ('Unable to sample any rows for the given conditions: '
```

Comment:
can we move this into the next if since it seems like it's only used if that condition is met

Reply:
It's used as a warning if `graceful_reject_sampling=True`